### ISSUE TITLE
Enhancement proposal: companions and summonable creatures

### ISSUE THREAD

**Issue by bw-mutley** [canismarko/dungeon-sheets#121]
Hi there, Mark. I got some days to work on this project, so I thought about some improvements. One of them is to include a compiled list of companions and summonable creatures in a similar way it is done to druid shapes. It would be a matter of including a few lines to `make_sheets.py`. 
Also, I would like to ask you which issues are real priorities, it seens like the issues list is not updated. Let me know how can I be of help.

**Comment by canismarko** [comment]
That sounds great! And I agree, should be straightforward. 

#88 could use a fix. #102 would be good, but I don't know if it gets used much. I really would like to get #75, but I don't know if a solution to that one exists.

Thanks for your efforts.

**Comment by bw-mutley** [comment]
Hi Mark, I've just made 4 pull requests - sorry if it is 'messy', I don't know how to make just a single one since the changes were in different folders. I've included a small report of changes in one of the commits. I've tested them using .tex and .html forms, but not epub. Let me know if any issues.

**Comment by canismarko** [comment]
@bw-mutley yeah we just had a baby so things have been hectic. Can you let me know which of the 4 PR's I should look at? I can work on it while baby is napping on me at some point.

Thanks!

**Comment by bw-mutley** [comment]
Hey Mark! Congratulations for the baby!
As for the pull requests: I did it wrong, there were changes in 4 folders and I didn't know how to proceed so ended making 4 requests. I guess we can cancel and I pull a single one including other things I've done after this one. Let me know what is best and I will organize things here.

**Comment by canismarko** [comment]
Thanks!

I'd prefer to have 1 PR, thanks.
